docs/README.md

```diff
@@ -1,29 +1,232 @@
-# User Guide
+# Duke Chatbot
+Duke chatbot is a java program that helps you to manage your tasks
 
-## Features 
+## Usage
 
-### Feature-ABC
+### `list` - displays a list of all tasks
 
-Description of the feature.
+Use this command when you want a list of all tasks you have added so far.
 
-### Feature-XYZ
+Example of usage: 
 
-Description of the feature.
+`list`
 
-## Usage
+Expected outcome:
+```
+  n____n
+ ( o v o )
+--u-----u---------------------------------
+-> Here are the tasks in your list:
+    1.[T][x] test1
+    2.[E][ ] test2  (at: Mon 2-4pm)
+    3.[D][ ] test5  (by: June 6th)
+```
+Description of the outcome.
+```
+Duke lists command in a chronogical order based on time you add the tasks.
+The first box of the task indicates the type of task (todo, event, deadline). 
+The second box indicates the completion. This is followed by the description of a task and date.
+```
```

Comment:
Very nice and clean. Good summary, makes it easier to understand what to expect.